docs/concepts/services-networking/manging-hosts-file-with-host-aliases.md

```diff
@@ -0,0 +1,71 @@
+---
+assignees:
+- rickypai
+- thockin
+title: Manging Hosts File with HostAliases
+redirect_from:
+- "/docs/user-guide/manging-hosts-file-with-hostaliases/"
+- "/docs/user-guide/manging-hosts-file-with-hostaliases.html"
+---
+
+* TOC
+{:toc}
+
+kubelet [manages](https://github.com/kubernetes/kubernetes/issues/14633) the hosts file for each container of the Pod. This is to prevent Docker from [modifying](https://github.com/moby/moby/issues/17190) the file after the containers have already been started.
```

Comment:
I'm not sure the historical context is necessary for documentation...or at least in the intro section.